src/microbe_stage/Microbe.Interior.cs

```diff
@@ -561,8 +561,8 @@ private void HandleReproduction(float delta)
 
             // We are in G1 phase of the cell cycle, duplicate all organelles.
 
-            // Except the nucleus
-            if (organelle.Definition.InternalName == "nucleus")
+            // Except the nucleus and the binding agent
+            if (organelle.Definition.InternalName is "nucleus" or "bindingAgent")
```

Comment:
Should use OrganelleDefinition.Unique in order to work with any organelle and not depend on the names.

Reply:
it checked the name before so i assumed that was the correct way.

Reply:
It was kind of fine for just a one organelle, but we now have this correct mechanism for defining any number of organelles as being placeable just once. So it should be used here.